pandas/tests/test_nanops.py

```diff
@@ -1004,6 +1004,95 @@ def prng(self):
         return np.random.RandomState(1234)
 
 
+class TestNumpyNaNFunctions(object):
```

Comment:
this can be a single function that is parameterized over all of the methods

Reply:
Sure, will do this soon - could you help with why the build is failing?
From looking at CircleCI it seems like it's not recognizing that [np.nanprod](https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.nanprod.html) is valid, do I need to remove the nanprod case for compat or something? Sorry if I'm being dense here.

Reply:
you might need to skip for older versions of numpy 

not sure when certain ones were added